pyqtgraph/graphicsItems/PlotDataItem.py

```diff
@@ -586,12 +621,32 @@ def getData(self):
                     y1[:,0] = y2.max(axis=1)
                     y1[:,1] = y2.min(axis=1)
                     y = y1.reshape(n*2)
-                
-                    
+
             self.xDisp = x
             self.yDisp = y
         return self.xDisp, self.yDisp
 
+    def dataRect(self):
+        """
+        Returns a bounding rectangle (as QRectF) for the full set of data
```

Comment:
If this function can return `None`, that should be indicated in the docstring (as well as a description of why that might be the case)